packages/flutter/lib/src/material/theme.dart

```diff
@@ -18,13 +18,15 @@ const Duration kThemeAnimationDuration = const Duration(milliseconds: 200);
 ///
 ///  * [AnimatedTheme]
 ///  * [ThemeData]
+///  * [MaterialApp]
 class Theme extends InheritedWidget {
   /// Applies the given theme [data] to [child].
   ///
   /// The [data] and [child] arguments must not be null.
   Theme({
     Key key,
     @required this.data,
+    this.isShadowTheme: true,
```

Comment:
boolean arguments should when possible default to false (because `foo: false` looks like something that could be safely removed without effect). Maybe `isPrimaryTheme` with the reverse semantics? Or `isMainTheme`? `isMaterialAppTheme`?